impl/src/planning/subplanners.rs

```diff
@@ -245,12 +245,15 @@ impl<'planner> WorkspaceSubplanner<'planner> {
     }
 
     // Additionally, the binary dependencies need to have their checksums added as well
-    for (pkg, info) in self.settings.binary_deps.iter() {
-      let version = semver::Version::parse(info.req())?;
-      package_to_checksum.insert(
-        (pkg.clone(), version.clone()),
-        fetch_crate_checksum(&self.settings.index_url, pkg, &version.to_string())?,
-      );
+    if self.settings.genmode == GenMode::Remote {
+      for (pkg, info) in self.settings.binary_deps.iter() {
+        let version = semver::Version::parse(info.req())?;
+        let key = fetch_crate_checksum(&self.settings.index_url, pkg, &version.to_string()).unwrap();
```

Comment:
Was there a reason `?` was changed to `unwrap()`?

Reply:
This was leftover from debugging. Sorry about that. Unintended change